postbuild.js

```diff
@@ -11,8 +10,10 @@ if (fs.existsSync(dest+'index.js')){
   fs.unlinkSync(dest+'index.js');
 }
 
+const mainFile = file.replace('/domain-search', './build');
```

Comment:
For reference, this change is required due to https://github.com/facebook/create-react-app/pull/2544